index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile-UI-project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="head">
+        <a href="#" class="logo"> Prince Rutagengwa</a>
```

Comment:
- [ ] You did a great job, your project is almost ready, please fix your logo top margin which is still looking different from the template, just add some space from the top:

Template:
![Captura de pantalla 2022-03-18 a la(s) 7 42 11 p  m](https://user-images.githubusercontent.com/83717285/159100029-40b248d5-18c4-49fd-8da1-5b6667233429.png)

Your design:
![Captura de pantalla 2022-03-18 a la(s) 7 44 41 p  m](https://user-images.githubusercontent.com/83717285/159100084-ef4f7ecb-f1f9-470c-8055-a6a0c7ae7652.png)